packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/testRunOperations/api/operations.ts

```diff
@@ -154,245 +156,22 @@ export async function _testRunDeserialize(
     throw createRestError(result);
   }
 
-  return {
```

Comment:
Why is this deserialization thing gone with this change?

Reply:
See this https://github.com/Azure/autorest.typescript/pull/2198#discussion_r1505222318.